rs/sns/governance/canister/governance_test.did

```diff
@@ -300,6 +300,12 @@ type Governance = record {
   neurons : vec record { text; Neuron };
   genesis_timestamp_seconds : nat64;
   target_version: opt Version;
+  timers : opt Timers;
```

Comment:
This should not be in the init arguments... but I guess we haven't split up the API crate from SNS yet...

Reply:
Agreed. So this isn't actionable yet.